package.json

```diff
@@ -61,7 +59,8 @@
     "rules": {
       "import/prefer-default-export": "off",
       "import/no-unassigned-import": "off",
-      "import/no-useless-path-segments": "off"
+      "import/no-useless-path-segments": "off",
+      "no-console": "off"
```

Comment:
Any reason why you globally disable `no-consol`-rule?

Reply:
I did and it was annoying.

Reply:
Once of those things where I'd like a different set of rules for local dev. Basically a set of rules for prod and dev. Because we do like to catch dangling debug statements but we do sometimes need them for debugging locally.